### ISSUE TITLE
Adding the duplicates dialogue to the climatic menu

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#4254]
We are going to be using the general duplicates dialogue this week in Lesotho.
For the climatic data the main check will be for daily data.  So having an adapted version in the climatic menu would be great.  In that case it could be initially to check for duplicate records where the Station and Date are the same - so this could be pre-filled.

A topic to discuss when I visit on 29th November.  We will check this with the general duplicates dialogue first.

**Comment by dannyparsons** [comment]
Was this discussed during the visit? Is anything going to happen on this for the next version?

**Comment by maxwellfundi** [comment]
Yes, this was discussed and I believe someone will be working on it to
implement the function for duplicates

**Comment by rdstern** [comment]
There is a small task now, instead of the special duplicates for climatic.  We have duplicates in the Prepare > Check data menu.  
Could there be a copy of this dialogue in the Climatic > Tidy and Examine menu.  It should go under Replace Values and before the horizontal line.

While there could you add the 2 one-variable summaries also to the bottom of the same menu.  The second is to be called **One Variable Graph** and the third is **One Variable Frequencies**.  They are the **Describe > One Variable > Graph** and **Describe > One Variable > Frequencies** dialogues
